views/auth/signup.hbs

```diff
@@ -0,0 +1,20 @@
+<h2>Signup</h2>
+<form action='auth/signup' method='post'>
```

Comment:
remember that here it's supposed to be /signup.

`<form action='/signup' method='post'>`